### DISCUSSION TITLE
Multiple users and copying PTS folder

### DISCUSSION THREAD

**Discussion by FazzaDazza** [Taxel/PlexTraktSync#946] [Support] [answered]
So I understand the easiest thing to do to manage multiple users sync, is to run multiple instances. 

What I would like to know, if I setup mine first and let it do the entire sync, then copy this for the next user. Do I just run the setup.bat and connect a different user. Also will it keep all the movie's/shows that its indexed or will it have to do it all over again ?

I have disabled collections and only use it for watched status tracking.  

Im doing this on windows. Ty in advance.

**Comment by glensc** [comment, marked as answer]
From [readme](https://github.com/Taxel/PlexTraktSync#setup):
> your credentials and API keys will be stored in `.env` and `.pytrakt.json` files.

so, you need to copy (rename) those files every time switching the user. or use environment variables to change their location.
